src/pretix/base/models/orders.py

```diff
@@ -1441,6 +1441,13 @@ def address_format(self):
         lines = [r.strip() for r in lines if r]
         return '\n'.join(lines).strip()
 
+    def requires_approval(self):
+        if self.item.require_approval:
+            return True
+        if self.variation is not None and self.variation.require_approval:
```

Comment:
Shouldn’t this be enough?

```suggestion
        if self.variation and self.variation.require_approval:
```

Reply:
P.S.: btw. good idea to move this to `AbstractPosition`!